lib/jsdom/level2/style.js

```diff
@@ -173,6 +173,13 @@ core.HTMLStyleElement._init = function() {
 
     evaluateStylesheet.call(this, content, this.sheet, this._ownerDocument.URL);
   });
+
+  this.addEventListener('DOMNodeRemovedFromDocument', function() {
```

Comment:
Ah alright, I just took the handlers from the LinkElement. Good to know! So I'll also remove the (anyway empty) handlers from there, too... it's misleading to other newcomers.